include/enrico/heat_fluids_driver.h

```diff
@@ -11,11 +11,23 @@ namespace enrico {
 //! Base class for driver that controls a heat-fluids solve
 class HeatFluidsDriver : public Driver {
 public:
-  explicit HeatFluidsDriver(MPI_Comm comm) : Driver(comm){};
+  explicit HeatFluidsDriver(MPI_Comm comm, double pressure) : Driver(comm), pressure_(pressure){};
 
-  //! Get the temperature in each element
-  //! \return Temperature in each element as [K]
+  virtual ~HeatFluidsDriver() = default;
+
+  //! Get the temperature in each region
+  //! \return Temperature in each region as [K]
   virtual xt::xtensor<double, 1> temperature() const = 0;
+
+  //! Get the density in each region
+  //! \return Temperature in each region as [g/cm^3]
+  virtual xt::xtensor<double, 1> density() const = 0;
+
+  //! States whether each region is in fluid
+  //! \return For each region, 1 if region is in fluid and 0 otherwise
+  virtual xt::xtensor<int, 1> fluid_mask() const = 0;
+
+  double pressure_;  //! System pressure in [MPa]
```

Comment:
Would it make sense to declare `pressure_` as an `xarray<double>`?  In Nek, the pressure would be treated a 0D container, and another surrogate could treat it as a higher-dimensional container.  xtensor seems to have pretty consistent semantics for this.

Reply:
If we move `pressure_` to `NekDriver` as in #31, then I think we can leave it as a scalar. 

If we add a pressure solution to `SurrogateHeatDriver`, then I don't think it's necessary to have the same pressure representation in `NekDriver` and `SurrogateHeatDriver` since `pressure_` would be private in `SurrogateHeatDriver`, similar to like `temperature_`. 

Nek's solution field is accessed by interface functions, while the `SurrogateHeatDriver`'s solution is made private so that we also are forced to access it through interfaces.

Reply:
That makes sense.  I'll move `pressure_` back to `NekDriver`.  I'd like to do it in this PR instead of your separate one (#31), since they both touch a lot of the same stuff.